packages/contracts-bedrock/test/universal/Specs.t.sol

```diff
@@ -34,6 +34,7 @@ contract Specification_Test is CommonTest {
         SYSTEMCONFIGOWNER,
         GUARDIAN,
         DEPUTYGUARDIAN,
+        PAUSEDEPUTY,
```

Comment:
oof 17 roles, that feels like a lot (not a blocker for this PR of course)

Reply:
Yep, welp